imap_processing/ialirt/process_ephemeris.py

```diff
@@ -72,6 +72,7 @@ def calculate_azimuth_and_elevation(
     altitude: float,
     observation_time: float | np.ndarray,
     target: str = SpiceBody.IMAP.name,
+    obsref: str = "IAU_EARTH",
```

Comment:
Wait, wait. If you are going to add this optional input, I think you should have the default be "ITRF93" so that others don't make the mistake of using the low accuracy frame.
```suggestion
    obsref: str = "ITRF93",
```

Reply:
Right, the reason is that I also use this code to predict ground station coverage, which doesn’t require high accuracy. That coverage routine relies on the science frame kernel, so I would’ve had to update the science frame kernel to include ITRF93 unnecessarily. But I will make ITRF93 the default, as suggested.